### PR TITLE
feat: add stubbed app-bridge

### PR DESCRIPTION
**Pull request by SamuelAlev**
Add `withAppBridgeStubs` and dummy test models (`Asset` and `User`)

### CODE REVIEW

File: src/types/User.ts [hunk 1]
```diff
@@ -1,10 +1,9 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```
Comment (SamuelAlev) [inline src/types/User.ts:1]: Cleaning unused properties

File: src/react/useBlockSettings.ts [hunk 2]
```diff
@@ -1,15 +1,17 @@
 import { useState } from 'react';
```
Comment (SamuelAlev) [inline src/react/useBlockSettings.ts:1]: Remove extra check for `blockId`

File: src/AppBridgeIframe.ts [hunk 3]
```diff
@@ -3,23 +3,13 @@
 import { generateRandomString } from './utilities/hash';
```
Comment (SamuelAlev) [inline src/AppBridgeIframe.ts:3]: Clean typing

File: src/tests/withAppBridgeStubs.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+type useStubedAppBridgeProps = {
+    blockSettings?: Record<string, unknown>;
+    blockAssets?: Record<string, Asset[]>;
+    editorState?: boolean;
+    openAssetChooser?: () => void;
+    closeAssetChooser?: () => void;
+};
+
+const getStubedAppBridge = ({
```
Comment (NadiaPosch) [inline src/tests/withAppBridgeStubs.tsx:16]:
```suggestion
const getStubbedAppBridge = ({
```

File: src/tests/withAppBridgeStubs.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AppBridgeNativeDummy } from './AppBridgeNativeDummy';
+import { Asset } from '../types/Asset';
+import { IAppBridgeNative } from '../types/IAppBridgeNative';
+import { ComponentType } from 'react';
+
+type useStubedAppBridgeProps = {
```
Comment (NadiaPosch) [inline src/tests/withAppBridgeStubs.tsx:8]:
```suggestion
type useStubbedAppBridgeProps = {
```

Comment (SamuelAlev) [reply src/tests/withAppBridgeStubs.tsx]: Fixed

File: src/tests/withAppBridgeStubs.tsx (hunk 4 above)
Comment (SamuelAlev) [reply src/tests/withAppBridgeStubs.tsx]: Fixed

**Pull request merged**